### PR TITLE
RAD-155: Add 'basic' schema to 'wfi_mosaic'

### PR DESCRIPTION
**Pull request by nden**
Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)

Closes #

The changes in this PR correspond to RAD-155 changes in spacetelescope/rad#390.
**pyproject should be updated before merging**

**Checklist**
- [ ] Added entry in `CHANGES.rst` under the corresponding subsection
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] Passed romancal regression testing on Jenkins / PLWishMaster. Link: https://plwishmaster.stsci.edu:8081/job/RT/job/romancal/XXX/

### CODE REVIEW

File: src/roman_datamodels/maker_utils/_common_meta.py [hunk 1]
```diff
@@ -481,9 +481,10 @@ def mk_mosaic_meta(**kwargs):
     dict (defined by the wfi_mosaic-1.0.0 schema)
     """
 
-    meta = {}
+    meta = mk_basic_meta(**kwargs)
+    meta["basic"] = mk_mosaic_basic(**kwargs.get("meta", {}))
```
Comment (PaulHuwe) [inline src/roman_datamodels/maker_utils/_common_meta.py:485]:
```suggestion
    meta["basic"] = mk_mosaic_basic(**kwargs.get("basic", {}))
```

File: src/roman_datamodels/maker_utils/_datamodels.py [hunk 2]
```diff
@@ -174,6 +174,7 @@ def mk_level3_mosaic(*, shape=(4088, 4088), n_images=2, filepath=None, **kwargs)
 
     wfi_mosaic = stnode.WfiMosaic()
     wfi_mosaic["meta"] = mk_mosaic_meta(**kwargs.get("meta", {}))
+    # wfi_mosaic["meta"]['basic'] = mk_mosaic_basic(**kwargs.get("meta", {}))
```
Comment (PaulHuwe) [inline src/roman_datamodels/maker_utils/_datamodels.py:177]:
```suggestion
    wfi_mosaic["meta"]['basic'] = mk_mosaic_basic(**kwargs.get("meta", {}))
```

Mosaic basic still exists.. it just lost some members.

**Review by PaulHuwe — CHANGES_REQUESTED** [review]

File: src/roman_datamodels/maker_utils/_common_meta.py (hunk 1 above)
Comment (nden) [reply src/roman_datamodels/maker_utils/_common_meta.py]: Thanks! That was the test issue, fixed now.

**Comment by nden** [comment]
@PaulHuwe Thanks for the review. This is ready now for final review.

**Review by PaulHuwe — APPROVED** [review]
LGTM.

**Pull request merged**